### PR TITLE
Refactored some utils (added `isValidPostalcode` and tests)

### PR DESCRIPTION
**Pull request by robinpiets**
Refactored the utils:
- added `isValidPostalcode` to utils. 
- updated `removeQuotes` 
- removed `uniqBy` since it was unused
- added tests for `removeQuotes` and `isValidPostalcode`

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

**Review by svenjens — APPROVED** [review]

File: apps/client/src/utils/index.js [hunk 1]
```diff
@@ -37,35 +37,10 @@ export const getRestrictionByTypeName = (restrictions, typeName) =>
 
 // IMTR helper
 export const removeQuotes = (str) =>
-  typeof str === "string" ? str.replace(/['"]+/g, "") : "";
+  typeof str === "string" ? str.replace(/['"]+/g, "") : str;
```
Comment (afjlambert) [inline apps/client/src/utils/index.js:40]: This does not make it easier to convert to typescript. I'm not sure why you even want to do this. It's also no a str anymore.

File: apps/client/src/utils/index.js [hunk 2]
```diff
@@ -37,35 +37,10 @@ export const getRestrictionByTypeName = (restrictions, typeName) =>
...
-        return map;
-      }
-
-      return map.has(key) ? map : map.set(key, item);
-    }, new Map())
-    .values();
-
-  return [...pickedObjects];
-};
-
```
Comment (afjlambert) [inline apps/client/src/utils/index.js:68]: Why did you remove this?

File: apps/client/src/utils/index.test.js [hunk 3]
```diff
@@ -85,4 +87,31 @@ describe("util", () => {
...
   });
+  test("removeQuotes", () => {
+    expect(removeQuotes('"19-c"')).toBe("19-c");
+    expect(removeQuotes('"Abc, 123. "')).toBe("Abc, 123. ");
+    expect(removeQuotes('" A-b,c.d/e{f}[g]+-h& 1 "')).toBe(
+      " A-b,c.d/e{f}[g]+-h& 1 "
+    );
+    expect(removeQuotes("text")).toBe("text");
+    expect(removeQuotes('"text')).toBe("text");
+    expect(removeQuotes(99)).toBe(99);
```
Comment (afjlambert) [inline apps/client/src/utils/index.test.js:98]: Hmmmm. 🤔

File: apps/client/src/utils/index.js [hunk 4]
```diff
@@ -90,3 +65,16 @@ export const scrollToRef = (ref, offset = 0) =>
...
   str && str.toLowerCase().replace(/[^a-z0-9]+/gi, "");
+
+/**
+ *
+ * This function return `true` or `false` wheter or not a value is a equal to a Dutch postalcode (eg: 1055XD)
+ *
+ * @param {string} value
+ */
+export const isValidPostalcode = (value) => {
+  const postalCodeRegex = /^[1-9][0-9]{3}[\s]?[A-Za-z]{2}$/i;
```
Comment (afjlambert) [inline apps/client/src/utils/index.js:76]: I'm not really a regex guru. But the `/i` is not needed or you can replace `[A-Za-z]` with `[a-z]` I think.

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/utils/index.js (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/utils/index.js:68]: `uniqBy` was unused

File: apps/client/src/utils/index.js (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/utils/index.js:40]: I agree it's not TypeScript proof, I can do that as well. But I would expect that if I run `removeQuotes(12)` that it returns `12` or `"12"`, but not `""`

Comment (afjlambert) [reply apps/client/src/utils/index.js:40]: I guess it should give a typescript error.
`12 is not a string.`
